tex/generic/pgf/frontendlayer/tikz/tikz.code.tex

```diff
@@ -2566,7 +2578,7 @@
 
 \def\tikz@skip#1{\tikz@scan@next@command#1}%
 \def\tikz@expand{%
-  \advance\tikz@expandcount by -1%
+  \advance\tikz@expandcount by -1\relax
```

Comment:
Could have also just left the EOL
```suggestion
  \advance\tikz@expandcount by -1
```

Reply:
Yes, this one is optional. Will remove it since half of the 6 matches returned by `grep -rni '\\advance\\tikz@expandcount' tex` are not ending with `\relax`.

Reply:
Oops, this is gone now, but the `%` at the end of the line is actually wrong.  It should be
```latex
\advance\tikz@expandcount by -1
```

Reply:
> Could have also just left the EOL

Apparently I misunderstood your first comment.